core/src/main/java/com/alipay/sofa/common/log/MultiAppLoggerSpaceManager.java

```diff
@@ -275,23 +276,23 @@ private static AbstractLoggerSpaceFactory createILoggerFactory(SpaceId spaceId,
                                   + SOFA_MIDDLEWARE_LOG_DISABLE_PROP_KEY + "=true");
             return NOP_LOGGER_FACTORY;
         }
-        // set log props
-        LogEnvUtils.processGlobalSystemLogProperties();
+
+        // set global system properties
+        spacesMap.get(spaceId).putAll(LogEnvUtils.processGlobalSystemLogProperties());
 
         // do create
         try {
             if (LogEnvUtils.isLogbackUsable(spaceClassloader)) {
                 String isLogbackDisable = System
                     .getProperty(LOGBACK_MIDDLEWARE_LOG_DISABLE_PROP_KEY);
-                if (isLogbackDisable != null
-                    && Boolean.TRUE.toString().equalsIgnoreCase(isLogbackDisable)) {
+                if (Boolean.TRUE.toString().equalsIgnoreCase(isLogbackDisable)) {
                     ReportUtil.reportWarn("Logback-Sofa-Middleware-Log is disabled!  -D"
                                           + LOGBACK_MIDDLEWARE_LOG_DISABLE_PROP_KEY + "=true");
                 } else {
-                    ReportUtil.report("Actual binding is of type [ " + spaceId.toString()
-                                      + " Logback ]");
+                    ReportUtil.reportDebug("Actual binding is of type [ " + spaceId.toString()
+                                           + " Logback ]");
                     LoggerSpaceFactoryBuilder loggerSpaceFactory4LogbackBuilder = new LoggerSpaceFactory4LogbackBuilder(
-                        spacesMap.get(spaceId));
+                        spaceId, spacesMap.get(spaceId));
```

Comment:
use local variable to save `spacesMap.get(spaceId)`, avoid do `get` many times.

Reply:
it just cares about key， wont fetch the value.